src/routes/About.tsx

```diff
@@ -19,6 +19,9 @@ export default function About(): JSX.Element {
         </a>{' '}
         mit dem Stand vom 12.01.2022.
       </p>
+      <p>
+        Ein Test wird mit einem grünen Harken angezeigt, wenn er Infizierte mit sehr hoher Viruslast zu über 75% erkennt.
```

Comment:
Should be "Haken" not "Harken".

Reply:
You are right, sorry for my bad spelling. 😔